packages/lodestar-params/src/interface.ts

```diff
@@ -3,6 +3,16 @@
  */
 
 export interface IBeaconParams {
+
```

Comment:
Can you order these fields in the same order as in the spec yaml files?
Also making sure its in the same order in the ssz type definition.

Reply:
done!